prime-router/src/test/kotlin/cli/tests/CompareDataTests.kt

```diff
@@ -0,0 +1,135 @@
+package gov.cdc.prime.router.cli.tests
```

Comment:
Ah, yes, testing that the test code is correct.     Excellent.

Now you need to write CompareDataTestsTests... ;)

Reply:
I know.  I had to.  I really was not feeling it doing manual testing and found bugs once I added these unit tests.